src/components/MultiGesture.tsx

```diff
@@ -216,6 +230,11 @@ class MultiGesture extends React.Component<MultiGestureProps> {
             y: gestureState.moveY,
           }
           this.props.onEnd?.({ sequence: this.sequence, clientStart: this.clientStart!, clientEnd, e })
+          if (Capacitor.isNativePlatform()) {
+            for (let i = 0; i < this.sequence.length; i++) {
+              Haptics.impact({ style: ImpactStyle.Light })
```

Comment:
Just to clarify, the onEnd haptics should be removed completely. The one impact per swipe is already achieved by onGesture above.

Reply:
Gotcha, I can do that. Thank you